### DISCUSSION TITLE
请问如何自定义网站的Favico

### DISCUSSION THREAD

**Discussion by GoldBro233** [pengzhanbo/vuepress-theme-plume#381] [Q&A] [answered]
- 我的网站此时无法自定义网站的Favico，我不知道在哪里定义这个。
- 我已经阅读了网站的手册，但是没有发现在哪里。
- 我已经尝试了仓库中对于config.ts中加入 head: []设置，但是报错

**Comment by pengzhanbo** [comment, marked as answer]
参考 本仓库中文档站点的配置： [config.ts](https://github.com/pengzhanbo/vuepress-theme-plume/blob/main/docs/.vuepress/config.ts#L22-L23)

**Reply by GoldBro233** [reply]
感谢您已经顺利解决！
